docs/maintaining/long-running-environment.md

```diff
@@ -0,0 +1,29 @@
+# cf-for-k8s long-running environment (LRE)
+
+The general idea for our long-running environment is to run it similarly to a production cf-for-k8s, and to measure useful uptime measurements (SLIs) that we can share with stakeholders about the stability of the platform.
+
+## Normal operation
+
+### Nightly Upgrades
+
+The [long-lived-env section of the cf-for-k8s stability pipeline](https://release-integration.ci.cf-app.com/teams/main/pipelines/cf-for-k8s-stability-tests?group=long-lived-env) houses our pipeline jobs that are responsible for deploying and upgrading the LRE. Every weeknight, the environment is automatically updated to the latest HEAD of main.
+
+### SLIs / Recurring Measurements
+
+see https://github.com/cloudfoundry/cf-for-k8s/blob/develop/docs/platform-availability.md#availability-during-normal-operation, which explains the SLIs and includes some SLI results, at least from the weeks leading up to the 1.0 release.
```

Comment:
It feels like this could be a relative link.

Reply:
I was having trouble coming up with any time that I'd want the relative link instead of the very latest. Do you have any examples?

Reply:
It feels like good practice to keep all links within the docs relative. If someone happens to be deliberately looking at a particular version of the docs (for example, `main` or a version tag), it seems strange to redirect them to develop in a non-obvious way.

I am also not sure how this will work when we implement a tool to convert the docs for the website.